Essentials/src/main/resources/plugin.yml

```diff
@@ -79,15 +79,15 @@ commands:
     aliases: [ecartographytable, carttable, ecarttable]
   clearinventory:
     description: Clear all items in your inventory.
-    usage: /<command> [player|*] [item[:<data>]|*|**] [amount]
+    usage: /<command> [(<player>|*)] [(<item>|*|**) [<amount>]]
     aliases: [ci,eci,clean,eclean,clear,eclear,clearinvent,eclearinvent,eclearinventory]
   clearinventoryconfirmtoggle:
     description: Toggles whether you are prompted to confirm inventory clears.
     usage: /<command>
     aliases: [eclearinventoryconfirmtoggle, clearinventoryconfirmoff, eclearinventoryconfirmoff, clearconfirmoff, eclearconfirmoff, clearconfirmon, eclearconfirmon, clearconfirm, eclearconfirm]
   condense:
     description: Condenses items into a more compact blocks.
```

Comment:
While you're at it changing the plugin.yml could you also correct this typo here?
I'm sure the EssX team would be happy about it.